fastatophy/fastatophy.go

```diff
@@ -0,0 +1,74 @@
+// Copyright ©2017 The bíogo Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// fastatophy converts a multiple-sequence alignment in FASTA to
+// PHYLIP (sequential) format.
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"os"
+
+	"github.com/biogo/biogo/alphabet"
+	"github.com/biogo/biogo/io/seqio"
+	"github.com/biogo/biogo/io/seqio/fasta"
+	"github.com/biogo/biogo/seq/linear"
+)
+
+var (
+	inf  = flag.String("inf", "test.aln", "input FASTA filename")
+	outf = flag.String("outf", "test.phy", "output PHYLIP filename")
+	help = flag.Bool("help", false, "help prints this message.")
+)
+
+func main() {
+	flag.Parse()
+	if *help {
+		flag.Usage()
+		os.Exit(0)
+	}
+
+	in, _ := os.Open(*inf)
```

Comment:
Unlike the other three examples, `fastatophy' does not
accept files from stdin or write output to stdout.I have
added error checking to inf and outf that was present
in an earlier version. I had taken them out because you
mentioned it as `dead code', but they probably are needed?